mobile/src/main/java/ch/epfl/sdp/mobile/state/StatefulSettingsScreen.kt

```diff
@@ -19,22 +20,29 @@ class AuthenticatedUserProfileScreenState(private val user: AuthenticatedUser) :
   override val name = user.name
   override val emoji = user.emoji
   override val followed = user.followed
+  val onEditProfileNameClickAction by onEditProfileNameClickAction
```

Comment:
Visibility isn't appropriate imho.

```suggestion
  private val onEditProfileNameClickAction by onEditProfileNameClickAction
```